package.json

```diff
@@ -8,7 +8,8 @@
   "source": "src/index",
   "files": [
     "src",
-    "lib"
+    "lib",
+    "addons"
```

Comment:
remove this, since we already moved `addons` into `src`

Reply:
updated